packages/sdks/typescript/src/index.ts

```diff
@@ -7,6 +7,7 @@ type Message = {
   content: string
 }
 type ChainCallResponse = {
```

Comment:
These types are repeated. We need to do a common shared package to share types and constants common to 

1. Web
2. Gateway
3. SDK
4. Compiler

All these packages should use this common package that would have only the constants and the base types.